lib/logstash/bundler.rb

```diff
@@ -0,0 +1,125 @@
+require "bundler"
```

Comment:
I meant specifically that we should isolate any monkeypatches into a special directory structure so we can more easily understand what patches we have enabled.

In the past, I've had many files in `lib/logstash/monkeypatches_for<reason>.rb` for bug fixes, performance fixes, or debugging. I'd love to unify that into just `lib/logstash/patches/<component>.rb` - This will, imo, make it more explicit where our monkeypatches are and where they are used (via `require` etc.)